core/src/main/java/com/linecorp/armeria/client/ClientFactoryOption.java

```diff
@@ -179,6 +179,12 @@
     public static final ClientFactoryOption<Long> IDLE_TIMEOUT_MILLIS =
             define("IDLE_TIMEOUT_MILLIS", Flags.defaultClientIdleTimeoutMillis());
 
+    /**
+     * The max-age of a socket connection in milliseconds.
+     */
+    public static final ClientFactoryOption<Long> MAX_AGE_MILLIS =
```

Comment:
- `MAX_CONNECTION_AGE_MILLIS`
- Could you add: `The connection is closed after this amount of time of creation.`